### PR TITLE
 Detect devices for existing brands: Xiaomi, TCL, Alcatel, Hisense, Huawei, Digma, Irbis, Cricket, Hyundai, Apple, Lenovo,  AT&T, LG, Motorola, Kyocera, Lanix, OnePlus, iPro, Dell, Zuum, Polaroid, CHIA and Detect library Akka HTTP

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: regexes/client/browsers.yml [hunk 1]
```diff
@@ -1166,6 +1166,11 @@
   version: '$1'
   engine:
     default: 'Blink'
+- regex: 'Opera%20Touch/(\d+[\.\d]+)? CFNetwork/.+Darwin/.+(?!.*x86_64)'
```
Comment (sgiehl) [inline regexes/client/browsers.yml:1169]: Shouldn't this be enough? Or do you match the other parts for the Engine?

```suggestion
- regex: 'Opera%20Touch/(\d+[\.\d]+)?'
```

File: regexes/oss.yml [hunk 2]
```diff
@@ -649,6 +657,10 @@
   name: 'iOS'
   version: ''
 
+- regex: 'iOS; (?!\d+)'
```
Comment (sgiehl) [inline regexes/oss.yml:660]: What is the `(?!\d+)` good for?

File: regexes/client/browsers.yml (hunk 1 above)
Comment (sanchezzzhak) [reply regexes/client/browsers.yml:1169]: this rule is only for iOS mobile devices, where the browser engine is always webkit

Comment (sgiehl) [reply regexes/client/browsers.yml:1169]: ok 👍

File: regexes/oss.yml (hunk 2 above)
Comment (sanchezzzhak) [reply regexes/oss.yml:660]: I'll redo everything.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**